app/controllers/game.js

```diff
@@ -330,6 +330,16 @@ export default Ember.Controller.extend({
     },
 
     /**
+     * Tap or untap a card.
+     *
+     * @param {GameCard} gameCard
+     */
+    tap: function (gameCard) {
+      gameCard.toggleProperty('isTapped');
+      this.get('model').save();
```

Comment:
The cool kids are doing it! 😎
On Tue, Jun 23, 2015 at 11:17 PM Zach McGonigal <notifications@github.com>
wrote:

> In app/controllers/game.js
> <https://github.com/SirZach/bbbbbbbbbbbbbbb/pull/163#discussion_r33114511>
> :
>
> > @@ -330,6 +330,16 @@ export default Ember.Controller.extend({
> >      },
> >
> >      /**
> > +     * Tap or untap a card.
> > +     *
> > +     * @param {GameCard} gameCard
> > +     */
> > +    tap: function (gameCard) {
> > +      gameCard.toggleProperty('isTapped');
> > +      this.get('model').save();
>
> I appreciate you modifying the game card on the controller. Data down,
> actions up!
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/SirZach/bbbbbbbbbbbbbbb/pull/163/files#r33114511>.
>